includes/features/SU-SWS/stanford_metatag_nobots/stanford_metatag_nobots.feature

```diff
@@ -1,16 +1,15 @@
-@stanford
 Feature: Stanford MetaTag NoBots
   In order to ensure that upgrades do not break existing functionality
   As an administrative user
   I want to ensure that the Stanford MetaTag NoBots module is working properly
 
-  @api @dev @safe
+  @api @safe
   Scenario: Stanford MetaTag NoBots
```

Comment:
@kmakitan, 
On a live site, we _want_ robots to index the site, and _don't want_ "noindex,nofollow,noarchive." Please make this scenario @dev.
We do need a _launch_ test that checks that we _don't_ see these. Please make this test.